src/proto.c

```diff
@@ -22,6 +22,7 @@
 
 #include "bouncer.h"
 #include "scram.h"
+#include "common/uthash_lowercase.h"
```

Comment:
This should not be added to each file, it should be isolated to `varcache.c` (or some other dedicated file). Any `HASH_` macros should only be used in that file, otherwise the hash algorithm differs between invocations of `HASH_`